### PR TITLE
fix(Loader): Fix loader-related bugs. Simplify loader logic. Delete L…

### PR DESCRIPTION
**Pull request by CorneeSean**
…oadingComponent base class in favor of explicit data fetching.

### CODE REVIEW

File: frontend/src/app/modules/interventions/edit-form/edit-form.component.ts [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
+    email: intervention.email,
+    status: intervention.status,
+    address: createAddress(intervention.address),
+  } as InterventionFormData);
 
-function transformToFormAddress(address: string) {
+const createAddress = (address: string) => {
   const addressParts = address.split(',').map(part => part.trim());
   return {
     street: addressParts[0] || '',
```
Comment (chodorowicz) [inline frontend/src/app/modules/interventions/edit-form/edit-form.component.ts]: nit: Good place to start using nullish coalescing

Comment (CorneeSean) [reply frontend/src/app/modules/interventions/edit-form/edit-form.component.ts]: We might start using it, but we would need to bump Typescript to 3.7.0 first. Our current version does not recognize the operator. Would you be willing to take care of this? :)

Comment (chodorowicz) [reply frontend/src/app/modules/interventions/edit-form/edit-form.component.ts]: Yeah, sure, I'll take it 👍

Comment (chodorowicz) [reply frontend/src/app/modules/interventions/edit-form/edit-form.component.ts]: https://github.com/lightning-team/Ekostraz/pull/57

**Pull request merged**